Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/PDFDiscretePresentationController.mm

```diff
@@ -1190,20 +1203,23 @@ static float elasticDeltaForTimeDelta(float initialPosition, float initialVeloci
     transform.scale(scaleFactor);
     transform.translate(centeringOffset.width(), centeringOffset.height());
 
-    m_rowsContainerLayer->setTransform(transform);
+    Ref rowsContainerLayer = *m_rowsContainerLayer.get();
+    rowsContainerLayer->setTransform(transform);
```

Comment:
Normally, we add a `protectedRowsContainerLayer()` which returns a Ref / RefPtr and call `protectedRowsContainerLayer()->setTransform(transform);`. There is no reason to introduce a local variable for a single function call.